meilisearch/client.py

```diff
@@ -13,7 +13,8 @@
 from meilisearch.config import Config
 from meilisearch.errors import MeiliSearchError
 from meilisearch.index import Index
-from meilisearch.task import get_task, get_tasks, wait_for_task
+from meilisearch.models.task import TaskInfo
+from meilisearch.task import cancel_tasks, get_task, get_tasks, wait_for_task
```

Comment:
This is working as is so just a suggestion here. At first glance it looked like `cancel_task` in `Client` was recursive which didn't make seance, but it isn't. Instead it is calling `cancel_task` from the `task` module and they just have the same name. You could alias the `task` functions on import and then call them by their alias to make that clear.

```suggestion
from meilisearch.task import cancel_tasks as cancel_task_func
from meilisearch.task import get_task as get_task_func
from meilisearch.task import get_tasks as get_tasks_func
from meilisearch.task import wait_for_task as wait_for_task_func
```

It's not a problem because for it to be recursive it would be `self.cancel_task` so ignore this if you don't think the alias makes things more clear.

Reply:
Hey @sanders41, I understand the need of your suggestion but the naming (I know this was just an example) makes it as much confusing.

What about creating a Task class with Static functions? My python is not the best, sorry if this is not a good practice/

```python
# In meilisearch/task.py
class Task: # should it be plural?

    @staticmethod
    def get_task(x):
        return x

    # etc... with the other methods
```

And then in this file:

```python
# In meilisearch/client.py
from meilisearch.task import Task

# ...

    def get_task(self, uid: int) -> Task:
       # ...
       task = Task.get_task(x)
```

Reply:
We already have a `Task` class in the templates so I don't think that's a good idea. I'll undo that naming and leave it as it was. Feel free to open a use issue on this if you think this part of the code should be changed

Reply:
Here is the issue: https://github.com/meilisearch/meilisearch-python/issues/604